Dockerfile

```diff
@@ -0,0 +1,33 @@
+# syntax=docker/dockerfile:1
+
+# Build
+FROM golang:1.21-alpine AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY *.go ./
+
+RUN go build -o /img-LinuxFr.org
+
+RUN go install golang.org/x/vuln/cmd/govulncheck@latest
+RUN govulncheck ./...
+RUN govulncheck --mode=binary /img-LinuxFr.org
+
+# Deploy
+FROM alpine
+ARG REDIS
+ENV REDIS=${REDIS:-redis:6379/0}
```

Comment:
I've just checked the difference between `ARG` and `ENV`: the first one is only used on Docker image build and the second one is used during build and runtime.

As the environment variable is not used during build, I would remove both lines.

Then I would replace the `ENTRYPOINT` with the default command line `CMD` (in _shell_ form):

```Dockerfile
CMD /img-LinuxFr.org -a 0.0.0.0:8000 -r ${REDIS:-redis:6379/0}
```
Furthermore we have to lisent on `0.0.0:8000` to be able to access the service from the host.

I'll add a commit on this branch with the proposed fixes.